lib/trade_tariff_backend.rb

```diff
@@ -50,6 +50,14 @@ def production?
       ENV["GOVUK_APP_DOMAIN"] == "tariff-management-production.cloudapps.digital"
     end
 
+    def staging?
+      ENV["GOVUK_APP_DOMAIN"] == "tariffs-uat.cloudapps.digital"
+    end
+
+    def development?
+      ENV["GOVUK_APP_DOMAIN"] == "tariffs-dev"
```

Comment:
should we expand that to be the whole domain?

Reply:
@matthewford so it's wrong env var in dev server env vars) not critical